CLAUDE.md

```
@@ -954,126 +456,66 @@ To find all models using this pattern: `grep -r "from_attributes.*True" src/omni
-
-6. **Use protocol names for DI**
-   ```python
-   event_bus = container.get_service("ProtocolEventBus")
-   ```
-
-7. **Use duck typing with protocols**
-   ```python
-   # Just use the protocol interface directly
-   service.publish(event)
-   ```
+1. **Always call super().__init__(container)**
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)